universe.go

```diff
@@ -75,6 +75,20 @@ func NewKafkaUniverse(ctx context.Context, logger Logger, envKeyPrefix string, c
 	return &res, nil
 }
 
+func NewUniverseWithSingleConsumer(ctx context.Context, logger Logger, envKeyPrefix string, confUnmarshal ConfigurationProvider, consumerID string, handler KafkaMessageHandler, contextInitializer KafkaContextInitializer, logEventRate int64) (*KafkaUniverse, error) {
+	kafkaUniverse, err := NewKafkaUniverse(ctx, logger, envKeyPrefix, confUnmarshal)
+	if err != nil {
+		return nil, err
+	}
+
+	if err = kafkaUniverse.InitializeConsumers(consumerID); err != nil {
```

Comment:
I suggest you to write a function with lines from 84 to 88 so you can call this function from NewUniverseWithSingleConsumer but also call it when you are using KafkaUniverse in a more common way